recipe/meta.yaml

```diff
@@ -27,6 +27,25 @@ requirements:
     - crosstool-ng >=1.23.0,<1.24
 
 outputs:
+  - name: ctng_sysroot_{{ ctng_target_platform }}
```

Comment:
What is this supposed to do?

Reply:
I am trying to get the sysroot link in one spot to be used later.

Reply:
Not sure if we need this in the end. Trying to make it work.

Reply:
Btw, we need to change `ctng_vendor` to `conda` instead of `conda_cos6` which should make the need for a symlink unnecessary.

Reply:
For sure, but I didn't want to break the current packages which have conda_cos6 lying around. It would be simpler to change that globally and then make a symlink later for compat. We can add the link to the sysroot cdt package.

Reply:
We can have the sysroot under `conda` and have symlinks for the binaries with `conda_cos6`.

Reply:
Yep exactly. This is what I meant above. I will push a change later to try this. Right now I want to let the current build go through.